src/main/java/com/revature/exceptions/InvalidStatusException.java

```diff
@@ -0,0 +1,20 @@
+package com.revature.exceptions;
+
+public class InvalidStatusException extends RuntimeException {
+
+	private static final long serialVersionUID = 1L;
+	
+	public InvalidStatusException(String arg0, Throwable arg1) {
```

Comment:
arg0 and arg1 are not very descriptive names for arguments. I would suggest "String message, Throwable cause".